src/material-experimental/mdc-list/nav-list.ts

```diff
@@ -26,11 +21,12 @@ import {MatInteractiveListBase, MatListBase} from './list-base';
   changeDetection: ChangeDetectionStrategy.OnPush,
   providers: [
     {provide: MatListBase, useExisting: MatNavList},
-    /**
-     * @deprecated Provider for `MatList` will be removed, use `MatNavList` instead.
-     * @breaking-change 11.0.0
-     */
-    {provide: MatList, useExisting: MatNavList},
   ]
 })
-export class MatNavList extends MatInteractiveListBase {}
+export class MatNavList extends MatListBase {
+  // An navigation list is considered interactive, but does not extend the interactive
+  // list base class. We do this because as per MDC, items of interactive lists are
+  // only reachable through keyboard shortcuts. We want all items for the navigation list
+  // to be reachable through tab key though. This ensures best accessibility.
```

Comment:
Similar comment here for a11y- there's actually probably a better pattern than we we do here, so I would remove "This ensures best accessibility."